### ISSUE TITLE
Why is it question with training.

### ISSUE THREAD

**Issue by DarkAlchy** [bmaltais/kohya_ss#1497]
I will preface this with I am no newb, or neophyte with training SD going all the way back to 1.4 but this LoRA stuff seems a bit hit or miss, and the LoCON stuff is 100% miss.  I can train a Dreambooth to perfection and use the extract to LoRA and perfect but I can't just make a LoRA as it almost always doesn't do anything.

What is the extraction doing that allows the LoRA to work that, apparently, I am not doing in the LoRA straight away training?  I am stumped.

I use Adafactor, though I used to use Prodigy, and all settings are the same between DB and the LoRA training.

**Comment by etha302** [comment]
I am more wondering if there are any other scripts to train lora or dreambooth at this point? I’ve given up on sd scripts/kohya mostly because of the slow performance and also as you said hit or miss situation.

**Comment by DarkAlchy** [comment]
Well, I never had a miss with 1.5 shiv's DB and I just had a miss with 6 trainings for DB in Kohya.  5 getting better and better as I adjusted the data and KERBLAMO, 100% failure.  I wish we had something else because I went digging last week and there is a whole lot of stuff out there that we simply do not have because we only have Kohya now.

I wish we had more too.  I wish I had the will to make this my own but I don't for several reasons and one is I really despise Python being a C++ person for decades.

I know this the scripts that were out there are all for 1.5 so left me out for 2.0/2.1 and, now, XL.

What is sad is the gui dev hasn't a clue about this either he just rolled a nice gui up for us instead of having to do all this all by hand, but all the real script stuff engine is 100% Kohya.
